src/components/dex/Trade.vue

```diff
@@ -357,14 +355,125 @@ export default {
       return this.side === 'Buy' ? this.percentForBuy(value) : this.percentForSell(value);
     },
 
-    percentForBuy() {
-      // TODO: fix this
-      console.log('return');
+    percentForBuy(value) {
```

Comment:
Is this method a straight copy from Desktop, or if not, what was changed (to make reviewing it faster)?